DigitalOffice/Services/CheckRightsService/Repositories/CheckRightsRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CheckRightsService.Database;
+using CheckRightsService.Database.Entities;
+using CheckRightsService.Repositories.Interfaces;
+using System.Collections.Generic;
+
+namespace CheckRightsService.Repositories
+{
+    public class CheckRightsRepository : ICheckRightsRepository
+    {
+        private readonly CheckRightsServiceDbContext dbContext;
+
+        public CheckRightsRepository(CheckRightsServiceDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+        public ICollection<Right> GetRightsList()
+        {
+            ICollection<Right> rights = (ICollection<Right>)dbContext.Rights;
```

Comment:
Do not send db model back, create new DTO model Right. Rename db model -> DbRight, Create mapper and map DbRight -> Right

Reply:
Does new directory Model have to be created? Or new DTO model and db model can both be in Entities directory?

Reply:
fixed